app/build.gradle

```diff
@@ -2,6 +2,7 @@ plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
     id 'io.github.takahirom.roborazzi'
+    id 'com.google.devtools.ksp'
```

Comment:
> Add the dependency to your module's build.gradle file. If you have a multi-module setup, add this dependency to all the modules with UI elements that should be displayed inside the Showkase browser.
Showkase supports both ksp and kapt. By default, it uses kapt as we only recently added ksp support.

https://github.com/airbnb/Showkase#installation